frontend/src/pages/projects/screens/detail/ProjectDetailsSidebar.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import {
+  JumpLinks,
+  JumpLinksItem,
+  Sidebar,
+  SidebarContent,
+  SidebarPanel,
+} from '@patternfly/react-core';
+import { ProjectSectionID, ProjectSectionTitle } from './types';
+
+const ProjectDetailsSidebar: React.FC = ({ children }) => {
+  const navItems = [
+    { id: ProjectSectionID.WORKSPACE, title: ProjectSectionTitle.WORKSPACE },
+    { id: ProjectSectionID.STORAGE, title: ProjectSectionTitle.STORAGE },
+    { id: ProjectSectionID.DATA_CONNECTIONS, title: ProjectSectionTitle.DATA_CONNECTIONS },
```

Comment:
With my other comments, this has to be less something you manually craft, and more something you use as a lookup map.

Reply:
I gave more details here: https://github.com/opendatahub-io/odh-dashboard/pull/578#discussion_r975795618